### PR TITLE
Test against PHP 7.3 and nightly

### PR DESCRIPTION
**Pull request by alcaeus**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | no    
| BC breaks?        | no     
| Related Issue     | 
| Need Doc update   | no

Adds PHP 7.3 and nightly to the build matrix. Failures are explicitly allowed for nightly, but it should help spot upcoming build issues.

@julienbourdeau are you cherry-picking PRs to master or would you like me to create a separate PR for it?

### CODE REVIEW

**Comment by alcaeus** [comment]
🙄

> friendsofphp/php-cs-fixer v2.13.1 requires php ^5.6 || >=7.0 <7.3 -> your PHP version (7.3.0RC5) does not satisfy that

@julienbourdeau personally, I would suggest dropping this: it's not checked in CI (which means that there are quite some errors in code) and as of 4 days ago, this was their opinion: 
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4123#issuecomment-442261343. If you want my opinion, use phpcs with a strict standard, e.g. `doctrine/coding-standard` our your own collection of sniffs from `slevomat/coding-standard`.

**Comment by alcaeus** [comment]
Build stable! 🎉 

At the very least, this fixes build issues for PHP 7.1 and 7.2 already present in master. Dropping php-cs-fixer (at least temporarily) lets us test against PHP 7.3. If you want to stick with php-cs-fixer, there are alternative ways to install that without blocking builds on PHP 7.3.

File: composer.json [hunk 1]
```diff
@@ -33,23 +33,20 @@
...
+        "symfony/proxy-manager-bridge": "*",
+        "symfony/yaml": "^3.4.0 || ^4.0.0"
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.0.x-dev"
+            "dev-master": "3.2.x-dev"
         }
     },
-    "scripts": {
```
Comment (nunomaduro) [inline composer.json]: @alcaeus What is the reason for removing this?

Comment (nunomaduro) [inline composer.json]: @alcaeus What is the reason for removing this?

**Pull request merged**